rotkehlchen/globaldb/asset_updates/manager.py

```diff
@@ -837,3 +674,51 @@ def _retrieve_update_files(
             }
 
         return updates
+
+    def apply_pending_compatible_updates(self) -> None:
+        """Apply any pending asset updates that are compatible with the current DB version
+        before upgrading to target_db_version.
+
+        This ensures we don't miss any asset updates that would become incompatible
+        after the DB upgrade.
+        """
+        try:
+            max_compatible_version = None
+            info_json = self._get_remote_info_json()
```

Comment:
Can we go with smaller try/except statements? This is the only thing that can raise RemoteError. So you can unindent the rest and catch the RemoteError. Log it properly and return